### PR TITLE
Gce

### PR DESCRIPTION
**Pull request by smkniazi**

### CODE REVIEW

File: templates/default/hdfs-site.xml.erb [hunk 1]
```diff
@@ -521,7 +521,7 @@
 <property>
    <name>dfs.cloud.provider</name>
    <value><%= node['hops']['cloud_provider'] %></value>
-   <description>Cloud provider. Currently only AWS is supported</description>
+   <description>Cloud provider. Suppoted cloud providers are AWS, GCS, AZURE</description>
```
Comment (maismail) [inline templates/default/hdfs-site.xml.erb:524]:
```suggestion
   <description>Cloud provider. Supported cloud providers are AWS, GCS, AZURE</description>
```

File: templates/default/hdfs-site.xml.erb [hunk 2]
```diff
@@ -646,4 +716,24 @@
...
+    <description>
+    </description>
+  </property>
+
+  <property>
+    <name>dfs.cloud.max.br.threads</name>
+    <value>10</value>
+    <description>
+    </description>
+  </property>
```
Comment (maismail) [inline templates/default/hdfs-site.xml.erb:738]: can you add these as attributes as well?

**Review by maismail — CHANGES_REQUESTED** [review]

File: templates/default/hdfs-site.xml.erb (hunk 2 above)
Comment (smkniazi) [reply templates/default/hdfs-site.xml.erb:734]: did it

File: templates/default/hdfs-site.xml.erb (hunk 1 above)
Comment (smkniazi) [reply templates/default/hdfs-site.xml.erb]: fixed

**Comment by smkniazi** [comment]
c

**Pull request closed without merging**